docker/README.md

```diff
@@ -0,0 +1,170 @@
+# red-pnda
+
+<img src="logo/pnda1r-trans.png" alt="Red PNDA logo" width="400" height="300"/>
+
+This framework provisions a minimal set of the PNDA ([pnda.io](http://pnda.io)) components to enable developers writing apps targeted at the full PNDA stack, to experiment with the PNDA components in a smaller, lightweight environment. Data exploration and app prototyping is supported using Jupyter and Apache Spark. 
+
+**Note**: 
+
+* Packages and application support isn't available on red-pnda. The respective tabs will **not** work on the PNDA console and will throw an error message.
+
+* This framework is not implemented with either scalability nor HA in mind and hence is unsuited for running production workloads. If this is a requirement, then one of the core PNDA flavors will be required - see PNDA [Guide](http://pnda.io/guide).
+
+
+The Red PNDA framework is intended as a platform for experimentation and is NOT formally supported at this point in time. Any issues encountered with the system can be reported to the standard PNDA support forums for informational purposes only.
+
+## Acknowledgement
+
+This work has been inspired by an initial concept created by Maros Marsalek ([https://github.com/marosmars](https://github.com/marosmars)) and Nick Hall ([https://github.com/cloudwiser](https://github.com/cloudwiser))
+
+## Prerequisites
+
+Tested with Ubuntu 18.04 distro.
+ 
+Docker Engine (tested with docker version **18.05.0-ce**)
+
+docker-compose (tested with docker-compose version **1.21.2**)
+
+
+Minimum amount of RAM / VCPU / Storage: TBD
```

Comment:
We know this is a key element which need clarification as users will face lot of issue on their environment due to missing vCPU, RAM or storage